src/gitgraph.js

```diff
@@ -363,13 +396,32 @@
       } else {
         commit.isMouseover = false;
       }
-    }
+    });
 
     if ( isOut ) {
       self.tooltip.style.display = "none";
     }
   };
 
+
+  /**
+   * Click event on commit dot
+   *
+   * @param {MouseEvent} event - Mouse event
+   *
+   * @self Gitgraph
+   **/
+  GitGraph.prototype.click = function ( event ) {
+    var self = this.gitgraph;
```

Comment:
@terhechte > I'm not sure that's still relevant since there is only one use of `self` here, at the same scope level.